### PR TITLE
feat: [CDB-119]: Fix for Dashboards test failure

### PR DESCRIPTION
**Pull request by Harness-JKhattak**
##### Summary:

- Fix for following error occurring occasionally when running the Dashboards test

```
FAIL src/modules/45-dashboards/pages/home/__tests__/Dashboards.test.tsx
  ● Dashboards › it should show edit dashboard form when editDashboard callback triggered

    TypeError: Cannot destructure property 'data' of 'custom_dashboards_1.useGetFolder(...)' as it is undefined.

      70 |
      71 |   const {
    > 72 |     data: foldersList,
         |           ^
      73 |     error,
      74 |     loading: isFoldersLoading
      75 |   } = useGetFolder({
```

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Review by vardanbansal-harness — DISMISSED** [review]

File: src/modules/45-dashboards/pages/home/__tests__/Dashboards.test.tsx [hunk 1]
```diff
@@ -97,10 +101,6 @@ describe('Dashboards', () => {
   })
 
   test('it should show edit dashboard form when editDashboard callback triggered', async () => {
-    jest
```
Comment (knagurski) [inline src/modules/45-dashboards/pages/home/__tests__/Dashboards.test.tsx:100]: Looks like down on line 156 you can do this as well. https://github.com/harness/harness-core-ui/blob/461677aeb263ac3d080793f909051cc8d6186e88/src/modules/45-dashboards/pages/home/__tests__/Dashboards.test.tsx#L156

**Comment by Harness-JKhattak** [comment]
retrigger jest

File: src/modules/45-dashboards/pages/home/__tests__/Dashboards.test.tsx [hunk 2]
```diff
@@ -152,9 +152,7 @@ describe('Dashboards', () => {
   })
 
   test('it should trigger clone callback when cloneDashboard triggered', async () => {
-    jest
-      .spyOn(customDashboardServices, 'useGetFolder')
-      .mockImplementation(() => ({ data: mockEmptyGetFolderResponse, loading: false } as any))
+    useGetFolderMock.mockReturnValue({ data: mockEmptyGetFolderResponse, loading: false } as any)
```
Comment (sean-harness) [inline src/modules/45-dashboards/pages/home/__tests__/Dashboards.test.tsx:155]: Do you need to mock the return value here when it is already done in the beforeEach? The only difference I see is `error: null` which shouldn't make a difference

**Review by knagurski — APPROVED** [review]
Nice and simple 👍🏻

**Pull request merged**